### PR TITLE
kitty: Update to 0.70.0.6 (and move off of fosshub)

### PR DESCRIPTION
**Pull request by rasa**
See #1454 and #1304

### CODE REVIEW

**Comment by Ash258** [comment]
Autoupdate will be broken due to multiple URLS (https://github.com/lukesampson/scoop/issues/2757)

**Comment by rasa** [comment]
I fixed it locally. I will remove from this manifest and push my fix in a new PR in scoop. Thanks for the heads up!

File: kitty.json [hunk 1]
```diff
@@ -26,11 +42,12 @@
         "Sessions",
         "kitty.ini"
     ],
+    "uninstaller": {
+        "script": "& taskkill.exe /im kageant.exe /f"
+    },
     "checkver": {
-        "url": "https://www.9bis.net/kitty/check_update.php?version=0.0.0.0",
+        "url": "http://www.9bis.net/kitty/check_update.php?version=0.0.0.0",
```
Comment (brandon93s) [inline kitty.json]: Back to `http`?

Comment (rasa) [reply kitty.json]: Seems `www.` supports HTTPS, but `kitty.` doesn't:
```shell
$ wget -S https://www.9bis.net
--2018-11-24 11:03:12--  https://www.9bis.net/
Resolving www.9bis.net (www.9bis.net)... 213.186.33.19
Connecting to www.9bis.net (www.9bis.net)|213.186.33.19|:443... connected.
HTTP request sent, awaiting response...
  HTTP/1.1 200 OK
  Set-Cookie: 60gpBAK=R1224225179; path=/; expires=Sat, 24-Nov-2018 20:23:26 GMT
  Date: Sat, 24 Nov 2018 19:03:13 GMT
  Content-Type: text/html
  Content-Length: 562
  Set-Cookie: 60gp=R4049309883; path=/; expires=Sat, 24-Nov-2018 20:04:15 GMT
  Server: Apache
  Accept-Ranges: bytes
  Vary: Accept-Encoding
  X-IPLB-Instance: 1051

$ wget -S https://kitty.9bis.net
--2018-11-24 11:03:42--  https://kitty.9bis.net/
Resolving kitty.9bis.net (kitty.9bis.net)... 213.186.33.5
Connecting to kitty.9bis.net (kitty.9bis.net)|213.186.33.5|:443... connected.
ERROR: no certificate subject alternative name matches
        requested host name 'kitty.9bis.net'.
To connect to kitty.9bis.net insecurely, use `--no-check-certificate'.

**Pull request merged**